src/evaluate.py

```diff
@@ -168,6 +168,12 @@ def write_preds(
             _write_wic_preds(
                 task, preds_df, pred_dir, split_name, strict_glue_format=strict_glue_format
             )
+
+        elif isinstance(task, GLUEDiagnosticTask):
+            # glue-diagnostic is caught above by being in ALL_GLUE_TASKS
```

Comment:
Wait, so what does this elif case catch? Just superglue?

Reply:
Currently yea